helm_deploy/use-of-force/templates/job.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
```

Comment:
@michalnawrockidj MAP-1636 ticket is for BVL and the narrative is something differnt than this UOF Cron Job recovery https://dsdmoj.atlassian.net/browse/MAP-1636